Proton/Sources/Core/PRTextStorage.m

```diff
@@ -153,71 +133,24 @@ - (void)setAttributes:(NSDictionary<NSString *,id> *)attrs range:(NSRange)range
     [self endEditing];
 }
 
-- (NSDictionary<NSAttributedStringKey,id> *)applyingDefaultFormattingIfRequiredToAttributes:(NSDictionary<NSAttributedStringKey,id> *)attributes {
-    NSMutableDictionary<NSAttributedStringKey,id> * updatedAttributes = attributes.mutableCopy;
-    if (!updatedAttributes) {
-        updatedAttributes = [[NSMutableDictionary alloc] init];
-    }
-
-    if (![attributes objectForKey:NSParagraphStyleAttributeName]) {
-        id value = _defaultTextFormattingProvider.paragraphStyle;
-        if (!value) {
-            value = [self defaultParagraphStyle];
-        }
-        [updatedAttributes setObject:value forKey:NSParagraphStyleAttributeName];
```

Comment:
`defaultTextFormattingProvider.paragraphStyle` and `self.defaultParagraphStyle` return two different types (`NSParagraphStyle` and `NSMutableParagraphStyle`). Type in `value` declaration could help ;) 
I am setting `NSParagraphStyle` for both cases now https://github.com/rajdeep/proton/pull/67/files#diff-af1fba35f4396b82d455305f01db4c3cR196